solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -165,12 +218,32 @@ function execute_deploy_script(){
 #format and mount system drive
 
 function mount_system_device(){
-    findmnt $SYSTEM_DRIVE && umount -l $SYSTEM_DRIVE
-    mkfs.ext4 -F $SYSTEM_DRIVE
-    mkdir -p /mnt/fs-local-volume
-    mount -t ext4 $SYSTEM_DRIVE $SYSTEM_DRIVE_MOUNT
-    mkdir -p /mnt/fs-local-volume/local-path-provisioner
-    sysctl -w vm.max_map_count=30000000
+    try
+    (
+        findmnt $SYSTEM_DRIVE && umount -l $SYSTEM_DRIVE
+        mkfs.ext4 -F $SYSTEM_DRIVE
+        mkdir -p /mnt/fs-local-volume
+        mount -t ext4 $SYSTEM_DRIVE $SYSTEM_DRIVE_MOUNT || throw $Exception
+        mkdir -p /mnt/fs-local-volume/local-path-provisioner
+        sysctl -w vm.max_map_count=30000000 || throw $Exception
+    )
+    catch || {
+    # handle excption
```

Comment:
Would it be better to use exception handling for every command instead of function.

Reply:
Yeah, it is command level exception handling only and not function level.